root/src/components/block/ProfileCardComponent.tsx

```diff
@@ -0,0 +1,145 @@
+import { Opacity } from "@mui/icons-material";
+import React, { useState } from "react";
+import styled from "styled-components";
+
+const Card = styled("div")`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 260px;
+  text-align: center;
+`;
+
+const SocialMediaArea = styled("div")`
+  text-align: center;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const MediaIcon = styled("img")`
+  width: 100%;
+  width: 1.3em;
+  margin: 6px 8px 4px 0px;
+  text-align: center;
+`;
+
+export interface ProfileProps {
+  image: string;
+  name: string;
+  positions: Array<string>;
+  socials?: Array<{ platform: string; link: string }>;
+  profileText?: string;
+  // mini?: boolean;
+}
+
+export const ProfileCard = ({
+  image,
+  name,
+  positions,
+  socials,
+  profileText,
+}: // mini,
+ProfileProps) => {
+  const [pic, setPic] = useState(true);
```

Comment:
Let's change the variable names here to make it clearer. 
From pic and setPic to showCard and setShowCard

Also please add typescript to the useState
useState<boolean>()

Reply:
> Also please add typescript to the useState
Nicholas, do you mean to change it into this `const [showCard, setShowCard] = React.useState<boolean>(false);`?

Reply:
Oh you dont need the React

So its just
useState<boolean>(false)

Reply:
🤣 Got it! Thank you!